### PR TITLE
Add a Web Development section, for now with the web-push library only

### PR DESCRIPTION
**Pull request by marco-c**
Not sure about the icon, so I just reused `layers`.

### CODE REVIEW

**Comment by marco-c** [comment]
Yeah I know, but I couldn't think of a better name. We could have a section specifically about Web Push, but then it would probably be too specific.
I wanted to make this a section containing libraries for web development (so for example this web-push, or something like localForage, and so on). Maybe Web Development Tools?

**Comment by djmitche** [comment]
@helfi92 @armenzg what do you think about that in relation to mozilla-frontend-infra (which is also in part a collection of libraries for web development)

**Comment by helfi92** [comment]
> Maybe Web Development Tools?

I like that name.

> what do you think about that in relation to mozilla-frontend-infra (which is also in part a collection of libraries for web development)

I can definitely see the repositories under mozilla-frontend-infra included in this new section but that could be in a separate PR.

File: src/data/webdevelopmenttools.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: Web Development Tools
+icon: wrench
```
[the remark this answers is not in the document]
Comment (marco-c) [reply src/data/webdevelopmenttools.yaml]: Done with 7d3a44050e2af48545da7cd0242bc1acb7b16284.

File: src/data/webdevelopmenttools.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: Web Development Tools
+summary: Tools and libraries for backend and frontend web development.
+icon: wrench
+repositories: 
+ - https://github.com/web-push-libs/web-push: good-first-bug
```
Comment (helfi92) [inline src/data/webdevelopmenttools.yaml]: This should be `web-push-libs/web-push: good-first-bug`.

**Pull request merged**